### ISSUE TITLE
Feature Request: ability to set "Enforce SSO-only authentication for this group"

### ISSUE THREAD

**Issue by cweidinger** [gitlabhq/terraform-provider-gitlab#518]
We're using terraform to ensure certain values are set so that no one can accidentally undo the setting. This is one of those critical security settings.

**Comment by teekhandekar** [comment]
@cweidinger Can I work on this ?

**Comment by timofurrer** [comment]
Checking the API docs it seems like this isn't supported in the API yet :'( 

@cweidinger did you have any specific API in mind which I'm missing here?

**Comment by jtymes** [comment]
> Checking the API docs it seems like this isn't supported in the API yet :'( 

Correct. [Related issue](https://gitlab.com/gitlab-org/gitlab/-/issues/297006).
